### PR TITLE
[terra-framework] Removing -ms-overflow-style css

### PR DESCRIPTION
**Pull request by BenBoersma**
### Summary
Removing the `-ms-overflow-style` css rule for the same reasons here https://github.com/cerner/terra-core/pull/2445.

Please add your name to the [CONTRIBUTORS.md] file. Adding your name to the [CONTRIBUTORS.md] file signifies agreement to all rights and reservations provided by the [License].

Please add an entry in the CHANGELOG.md file in each package you modify.

Thanks for contributing to Terra.
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

**Comment by BenBoersma** [comment]
Need to have a discussion on these removal, seems removing this in `terra-scroll` had unintended consequences

**Comment by BenBoersma** [comment]
As per scrum discussion, we're okay with the changes we're seeing, so we're good to move forward with this.

File: packages/terra-application-menu-layout/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Unreleased
 3.5.0 - (June 12, 2019)
 ------------------
 ### Removed
+* Removed '-ms-overflow-style' style
```
Comment (StephenEsser) [inline packages/terra-application-menu-layout/CHANGELOG.md]: Looks like this entry ended up in the wrong section, we should move it back into Unreleased

Comment (BenBoersma) [reply packages/terra-application-menu-layout/CHANGELOG.md]: Fixed https://github.com/cerner/terra-framework/pull/728/commits/f6bc73acb8c9ff28ad329fa9f653d80f186a7e7f and https://github.com/cerner/terra-framework/pull/728/commits/5899a38012b4d48daae0e8d5690426fd3dbbc88b

**Pull request merged**